src/test/java/org/isf/medicalsinventory/Tests.java

```diff
@@ -572,4 +575,63 @@ void testDeleteInventory() throws Exception {
 		assertThat(deletedInventory).isNotNull();
 		assertThat(deletedInventory.getStatus()).isEqualTo(InventoryStatus.canceled.toString());
 	}
+	
+	@Test
+	void testValidateMedicalInventory() throws Exception {
+		Ward ward = testWard.setup(false);
+		wardIoOperationRepository.saveAndFlush(ward);
+		MedicalInventory inventory = testMedicalInventory.setup(ward, false);
```

Comment:
you have to persist also chargeType and dischargeType as "inventory+" and "inventory-" are not found in the inmemory DB